lib/chat_api_web/controllers/registration_controller.ex

```diff
@@ -23,7 +23,7 @@ defmodule ChatApiWeb.RegistrationController do
             # # TODO: figure out what we want to do here -- it's not currently
             # # obvious that a user invitation expires after one use.
             # ChatApi.UserInvitations.expire_user_invitation(invite)
-            conn |> enqueue_welcome_email() |> send_api_token()
+            conn |> send_registration_event |> enqueue_welcome_email() |> send_api_token()
```

Comment:
nit - let's use parens here for consistency (i think it's generally encouraged to use parens even when they can be omitted)